lib/absinthe/phase/document/result.ex

```diff
@@ -15,6 +15,9 @@ defmodule Absinthe.Phase.Document.Result do
   defp process(blueprint) do
     result =
       case blueprint.execution do
+        %{validation_errors: [], result: nil} ->
```

Comment:
Seems sensible, can you create a test case that exercises this?

Reply:
Added a couple of tests; unsure of correct placement -- please advise better location.

Reply:
nm.